docker-environment-driver/src/main/resources/isolatedDockerConfiguration/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM fedora
+MAINTAINER Marek Novotny <mnovotny@redhat.com>
+
+# install java and zip
+RUN yum -y update && yum clean all
+RUN yum -y install tar gzip java java-devel && yum clean all
+
+# set JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-openjdk
+
+## jenkins instalation
+ENV JENKINS_HOME /var/jenkins_home
+
+# Jenkins is ran with user `jenkins`, uid = 1000
+# If you bind mount a volume from host/vloume from a data container,
+# ensure you use same uid
+RUN useradd -d "$JENKINS_HOME" -u 1000 -m -s /bin/bash jenkins
+# Jenkins home directoy is a volume, so configuration and build history
+# can be persisted and survive image upgrades
+VOLUME /var/jenkins_home
+# `/usr/share/jenkins/ref/` contains all reference configuration we want
+# to set on a fresh new installation. Use it to bundle additional plugins
+# or config file with your custom jenkins Docker image.
+RUN mkdir -p /usr/share/jenkins/ref/init.groovy.d
+COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-angent-port.groovy
+ENV JENKINS_VERSION 1.554.1
+# could use ADD but this one does not check Last-Modified header
+# see https://github.com/docker/docker/issues/8331
+RUN curl -L http://mirrors.jenkins-ci.org/war-stable/$JENKINS_VERSION/jenkins.war -o /usr/share/jenkins/jenkins.war
+ENV JENKINS_UC https://updates.jenkins-ci.org
+RUN chown -R jenkins "$JENKINS_HOME" /usr/share/jenkins/ref
+# for main web interface:
+EXPOSE 8080
```

Comment:
we need to tidy up jenkins installation anyway, I am not sold to have jenkins master in a default build image. we should have there only useful things in that image.